plugins/scaffolder-react/src/next/components/Stepper/ErrorListTemplate/errorListTemplate.tsx

```diff
@@ -39,9 +40,28 @@ const useStyles = makeStyles((_theme: Theme) =>
  *
  * @public
  */
-export const ErrorListTemplate = ({ errors }: ErrorListProps) => {
+export const ErrorListTemplate = ({ errors, schema }: ErrorListProps) => {
   const classes = useStyles();
 
+  function renderError(error: RJSFValidationError) {
```

Comment:
Maybe more appropriate to name it as `formatErrorMessage`? 

As it doesn't really render anything 😅

Reply:
Done!